proposals/0035-linalg-matrix.md

```diff
@@ -224,8 +224,12 @@ void WaveMatrixExample() {
   using MatrixAccum32Ty = Matrix<MatrixComponentType::F32, 8, 16,
                                  MatrixUse::Accumulator, MatrixScope::Wave>;
 
-  MatrixATy MatA = MatrixATy::Load(B, 0, 8 * 4, MatrixLayout::RowMajor);
-  MatrixBTy MatB = MatrixBTy::Load(B, 0, 32 * 4, MatrixLayout::RowMajor);
+  MatrixATy MatA = MatrixATy::Load(
+      B, 0, /* Row stride = number of columns * element size */ 32 * 4,
+      MatrixLayout::RowMajor);
+  MatrixBTy MatB = MatrixBTy::Load(
+      B, 0, /* Row stride = number of columns * element size */ 16 * 4,
+      MatrixLayout::RowMajor);
```

Comment:
FWIW, I find this really hard to read with the `/*` and the `*` and the `*/` all looking like they're a part of the expression.  It's slightly easier when it's rendered with syntax highlighting.